src/SqlPersistence.PersistenceTests/SqlPersistence.PersistenceTests.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net461;netcoreapp3.1</TargetFrameworks>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>$(SolutionDir)Test.snk</AssemblyOriginatorKeyFile>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\ScriptBuilder\ScriptBuilder.csproj" />
+    <ProjectReference Include="..\SqlPersistence\SqlPersistence.csproj" />
+    <ProjectReference Include="..\TestHelper\TestHelper.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.6.1" />
+    <PackageReference Include="NServiceBus.PersistenceTests.Sources" Version="7.4.0" />
+    <PackageReference Include="NUnit" Version="3.12.0" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.17.0" />
+    <PackageReference Include="Particular.Approvals" Version="0.2.0" />
+    <PackageReference Include="PublicApiGenerator" Version="9.3.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <!---  Remove this exclusion once PR is merged: https://github.com/Particular/NServiceBus/pull/5707 and remove When_worker_tries_to_complete_saga_update_by_another_pessimistic_tweaked.cs-->
+    <Compile Remove="\**\When_worker_tries_to_complete_saga_update_by_another_pessimistic.cs" />
+
+    <Compile Remove="\**\When_concurrent_update_exceed_lock_request_timeout_pessimistic.cs" />
```

Comment:
Do we need a comment about why this one is excluded?

Reply:
They are all documented in the issue, the only reason that I added a comment to the other one, is because that's actually already fixed, the package is just not yet available